pkg/operator/status.go

```diff
@@ -341,6 +341,7 @@ func getRelatedObjects() []configv1.ObjectReference {
 		{Group: "tuned.openshift.io", Resource: "profiles", Name: "", Namespace: tunedMf.Namespace},
 		{Group: "tuned.openshift.io", Resource: "tuneds", Name: "", Namespace: tunedMf.Namespace},
 		{Group: "apps", Resource: "daemonsets", Name: dsMf.Name, Namespace: dsMf.Namespace},
+		{Group: "performance.openshift.io", Resource: "performanceprofiles", Name: ""},
```

Comment:
Is this relevant to this fix?

In addition, the commit message looks confusing, could you please clean it up?
Not sure why @MarSik is listed as author

Reply:
It looks like a bad rebase, the commit message contains couple of other older commits too